src/main/java/randoop/main/GenTests.java

```diff
@@ -537,6 +597,34 @@ public boolean handle(String[] args) {
     return true;
   }
 
+  /**
+   * Tallies the number of sequences an operation occurs in.
+   *
+   * @param tallyMap the count map to increment for each operation
+   * @param sequences sequences to process for operations
+   */
+  private void tallyOperationsInSequences(
```

Comment:
Why does this method take an out-parameter and return void?
Methods are usually clearer if they do not modify their arguments, and I think it would be nicer for this to return a freshly-constructed map, unless there is a reason to prefer an out-parameter.